gdx/test/com/badlogic/gdx/utils/LongArrayTest.java

```diff
@@ -0,0 +1,425 @@
+/*
+ * Copyright 2024 See AUTHORS file
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.badlogic.gdx.utils;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class LongArrayTest {
+	/** Test of the different adding methods */
+	@Test
+	public void addTest () {
+		/*
+		 * Test of the classic add(long value) method, it should be adding a number at the first available place.
```

Comment:
At risk of repeating what I said at https://github.com/libgdx/libgdx/pull/7514#discussion_r1845546463, the comments throughout this seem more plentiful and verbose than they need to be. It's irrelevant whether a method is classic or otherwise, and there's a lot of "Test of the ... method" in here stating the obvious. Since multiple tests are performed in this test, it's not unreasonable to break it up with comments, but many would argue that unit tests are supposed to be as small as possible. Since they are run automatically, it doesn't matter much if there's loads of them.